nengo_spinnaker/simulator.py

```diff
@@ -171,12 +169,10 @@ def __init__(self, network, dt=0.001, period=10.0, timescale=1.0):
         ))
 
         logger.info("Setting router timeout to 16 cycles")
-        for x in range(machine_width):
-            for y in range(machine_height):
-                with self.controller(x=x, y=y):
-                    if (x, y) in system_info:
-                        data = self.controller.read(0xf1000000, 4)
-                        self.controller.write(0xf1000000, data[:-1] + b'\x10')
+        for x, y in system_info.chips():
```

Comment:
Just iterating over `system_info` is sufficient. (It is a dictionary {(x, y): chip_info}).